backend/clients/twilio_client.py

```diff
@@ -20,24 +21,30 @@
     Chat,
     User,
     Client as ClientModel,
+    Chat_Media
 )
-from typing import Optional
+from typing import Optional, List
 from pydub import AudioSegment
 from base64 import b64encode
 from sqlmodel import Session, select
 from core.database import engine
 from utils.util import get_value_or_raise_error, TextConverter
+from utils.storage import upload
 
 
 logging.basicConfig(level=logging.INFO)
 logger = logging.getLogger(__name__)
 
 MAX_WHATSAPP_MESSAGE_LENGTH = 1500
 STORAGE = "./storage"
+ALLOWED_MESSAGE_TYPES = ["text", "image"]
 
 
 def save_chat_history(
-    session: Session, conversation_envelope: dict, message_body=str
+    session: Session,
```

Comment:
I don't think this is the correct place to save the chat history. The code in this file should be specific to Twilio only and saving chat history is general to Slack and WhatsApp (and SMS later on). This method should be moved to a central place and not be on the Twilio handler.

Reply:
Yes, we can move this later.